circuit-lib/circuit-lib.js/src/elgamal-babyjubjub/elgamal.ts

```diff
@@ -0,0 +1,287 @@
+import { AffinePoint } from "@noble/curves/abstract/curve";
+import { ExtPointType } from "@noble/curves/abstract/edwards";
+import { babyJub as CURVE } from "./babyjubjubConfig";
+import { encode } from "./decode";
+import assert from "assert";
+import * as crypto from "crypto";
+
+const createBlakeHash = require("blake-hash");
+const ff = require("ffjavascript");
+const Scalar = ff.Scalar;
+
+type SnarkBigInt = bigint;
+type PrivKey = bigint;
+type PubKey = ExtPointType;
+type BabyJubAffinePoint = AffinePoint<bigint>;
+type BabyJubExtPoint = ExtPointType;
+
+/**
+ * A private key and a public key
+ */
+interface Keypair {
+  privKey: PrivKey;
+  pubKey: PubKey;
+}
+
+// The BN254 group order p
+const SNARK_FIELD_SIZE: SnarkBigInt = BigInt(
```

Comment:
I think a better name is ```BABY_JUB_JUB_FIELD_SIZE```.
Can't we import it from babyJubJubConfig.ts?

Reply:
Removed after replacing with imported function from [maci-crypto](https://www.npmjs.com/package/maci-crypto)